invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/doi.html

```diff
@@ -6,7 +6,8 @@
   it under the terms of the MIT License; see LICENSE file for more details.
 #}
 
-{%- set id_doi = metadata.identifiers|doi_identifier %}
+{%- set id_doi = record.pids|doi_identifier %}
+<dt><b>{{_('DOI')}}</b></dt>
```

Comment:
I would have added a `if doi is set` (pseudcode): DOI can be optional and might not be there. What happens in that case?
Maybe it is already handled somewhere else...

Reply:
Needs discussion IRL after tomorrow standup. Not sure we understand the expected end goal/result

Reply:
Discussed IRL: If no DOI show **nothing** (no title, no sentence)